server/src/config/index.ts

```diff
@@ -1,10 +1,47 @@
-import isDocker from 'is-docker';
+import dotenv from 'dotenv';
 import { IConfig } from './IConfig';
-import * as dev from '../../config/dev.json';
-import * as stage from '../../config/stage.json';
-import * as docker from '../../config/docker.json';
 
-const seedEnvironment = process.env.SEED_ENVIRONMENT || 'dev';
-const config : IConfig = isDocker() ? docker : seedEnvironment === 'dev' ? dev : stage; 
+dotenv.config();
```

Comment:
@Alexander-Wang-IPS Should this line go in `Server.ts`? Not sure. Just looking for possible redundancies with the configuration.